packages/desktop-client/src/components/schedules/ScheduleLink.tsx

```diff
@@ -88,18 +91,19 @@ export function ScheduleLink({
             }}
           >
             <Text>
-              Choose the schedule{' '}
               {ids?.length > 1
-                ? `these ${ids.length} transactions belong`
-                : `this transaction belongs`}{' '}
-              to:
+                ? t(
+                    `Choose the schedule these {{count}} transactions belong to:`,
+                    { count: ids.length },
+                  )
+                : t(`Choose the schedule this transaction belongs to:`)}
```

Comment:
Since we're using `count` now I don't think the `ids?.length > 1` ternary is needed at all. i.e. this block could be:

```tsx
<Text>
  <Trans count={ids.length}>
    Choose the schedule these {{count}} transactions belong to:
  </Trans>
</Text>
```

Reply:
@jfdoming Does it properly handle the case where `count` is `null`? Because `ids?.length` might return null.

Reply:
Also, I can't get your syntax to work.  This one does though:

```tsx
<Trans>
  Choose the schedule these {{ count: ids?.length }} transactions belong to:
</Trans>
```

Reply:
Ah, I didn't actually know that `ids?.length > 1` was a valid comparison if `ids` was null. As for the syntax, I realized for it to work you'd need to declare a variable called `count` in scope (following https://arc.net/l/quote/qtkroxss). Yours successfully builds too!—but there's a UI problem that now it displays the literal string "{{count}}". I think we can fix like this:
```tsx
<Text>
  <Trans count={ids?.length ?? 0}>
    Choose the schedule these {{ count: ids?.length }} transactions belong to:
  </Trans>
</Text>
```

Maybe the cleanest version is this:
```tsx
<Text>
  {t('Choose the schedule these {{count}} transactions belong to:', {count: ids?.length ?? 0})}
</Text>
```

Side note: I guess the type in `props` should be fixed to indicate that `ids` is in fact nullable.

Reply:
I guess the question is: is it nullable?  The code here does `ids?` in several places, but not *all* places.  And if the type of the call is not nullable then I don't think it can ever be passed in null?  Maybe these `ids?.` should just be normal accessors?

Reply:
The only instance of `<ScheduleLink` I could find was from `Modals.tsx` where it comes from `options?.transactionIds`. That implies it could be nullable, but maybe for correctness we want to assume it isn't? (Since ideally types should eliminate the need for `options?.transactionIds`.)

Reply:
I think for now, given the scope of this PR is translation, I will just leave is 'as is' and assume it is nullable.